@kiva/kv-components/src/vue/KvFlag.vue

```diff
@@ -40,13 +43,21 @@ export default {
 			type: String,
 			required: true,
 		},
+		/**
+		 * Name of the country of the flag
+		* */
+		name: {
```

Comment:
Will add this new prop break existing usages that relied on the `getNameByCode`?

Reply:
In other words, will it be a heavy lift to add this new `required` prop the next time this package is updated in the Vue apps?

Reply:
I don't expect it to be a heavy lift since in most cases the iso code data came from a graphql Country object, so the country name could be easily added to the query.

One case where it might be a heavy lift is the phone number input in the mfa settings in UI, where there is a long list of country codes that didn't come from a graphql call. We could handle that case by giving the phone number input component a country name list, and then it's only loaded when entering a phone number instead of every time a flag is used.

Reply:
Do you plan on tackling those changes after merging this? Luckily we aren't using `KvFlag` in CPS, where we are currently using the new side sheet and still making changes, but I could see us being requested to use the side sheet BP in MyKiva in the next few months.

Reply:
Yeah, Im planning on updating UI and CPS for all the breaking changes I introduced with this PR, including adding the country name to KvFlag usage.